sycl/doc/SpecializationConstants.md

```diff
@@ -197,96 +197,110 @@ and the user says
 
 ##### The SpecConstant pass changes
 
- - The SpecConstants pass in the post-link will have the following IR as input (`sret` conversion is omitted for clarity):
+The SpecConstants pass in the post-link will have the following IR as input
+(`sret` conversion is omitted for clarity):
 
 ```
-  %spec_const = call %struct.POD __sycl_getCompositeSpecConstantValue<POD type mangling> ("MyConst_mangled")
+%struct.POD = type { [2 x %struct.A], <2 x i32> }
+%struct.A = type { i32, float }
+
+%spec_const = call %struct.POD __sycl_getCompositeSpecConstantValue<POD type mangling> ("MyConst_mangled")
 ```
 
-where `__sycl_getCompositeSpecConstantValue` is a new "intrinsic"
- (in addition to `__sycl_getSpecConstantValue`) recognized by SpecConstants pass,
- which creates a value of a composite (of non-primitive type) specialization constant.
- It does not need a default value, because its default value consists of default
- valued of its leaf specialization constants (see below).
+`__sycl_getCompositeSpecConstantValue` is a new "intrinsic" (in addition to
+`__sycl_getSpecConstantValue`) recognized by the `SpecConstants` pass, which
```

Comment:
@kbobrovs, what do you think about renaming existing `__sycl_getSpecConstantValue` into `__sycl_getScalarSpecConstantValue` to unify names and add more clarity?

Reply:
@AlexeySachkov, I don't mind